packages/block-library/src/column/edit.js

```diff
@@ -69,11 +69,7 @@ function ColumnEdit( {
 			</InspectorControls>
 			<InnerBlocks
 				templateLock={ false }
-				renderAppender={ (
-					hasChildBlocks ?
-						undefined :
-						() => <InnerBlocks.ButtonBlockAppender />
-				) }
+				renderAppender={ ! hasChildBlocks && InnerBlocks.ButtonBlockAppender }
```

Comment:
Why did this change get made? Just rolling in another fix? Checking whether it is related and I missed something.

Reply:
I simply discovered that in the columns block while testing and it simply is better code.

Reply:
All good. Leave it in 👍